node/cmd/ccq/utils.go

```diff
@@ -39,3 +49,151 @@ func FetchCurrentGuardianSet(rpcUrl, coreAddr string) (*common.GuardianSet, erro
 		Index: currentIndex,
 	}, nil
 }
+
+type Config struct {
+	Permissions []User `json:"Permissions"`
+}
+
+type User struct {
+	UserName     string        `json:"userName"`
+	ApiKey       string        `json:"apiKey"`
+	AllowedCalls []AllowedCall `json:"allowedCalls"`
+}
+
+type AllowedCall struct {
+	EthCall *EthCall `json:"ethCall"`
+}
+
+type EthCall struct {
+	Chain           int    `json:"chain"`
+	ContractAddress string `json:"contractAddress"`
+	Call            string `json:"call"`
+}
+
+type Permissions map[string]*permissionEntry
+
+type permissionEntry struct {
+	userName     string
+	apiKey       string
+	allowedCalls allowedCallsForUser // Key is something like "ethCall:2:000000000000000000000000b4fbf271143f4fbf7b91a5ded31805e42b2208d6:06fdde03"
+}
+
+type allowedCallsForUser map[string]struct{}
```

Comment:
You can create a type with an empty struct?

Reply:
Yes, this is how you do a set in go.